.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+
```

Comment:
- [ ] Kindly include `test.md` in your .gitignore file and remove `test.md` file from the repo. The `test.md` file should be ignored as it's in the requirement for this project.

Reply:
- [ ] Hello @DD-code1 I can not see your `test.md` file in `.gitignore` file, kindly add it, you only deleted it from the repo.